internal/store/postgres/migrations/20230428170234_add_metaschema_table.up.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE IF NOT EXISTS metaschema (
```

Comment:
Lets also add ID, created_at, updated_at columns as well

Reply:
Earlier thought of adding it, but didn't find a need for the same. How will we use created_at and updated_at field. I thought if we are already updating the in-memory map while creating/ updating the schemas, these fields wouldn't be required.